runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttConditionConfigAdapter.java

```diff
@@ -89,6 +101,15 @@ public JsonObject adaptToJson(
             {
                 JsonObjectBuilder publishJson = Json.createObjectBuilder();
                 publishJson.add(TOPIC_NAME, p.topic);
+                if (!p.params.isEmpty())
+                {
+                    JsonObjectBuilder params = Json.createObjectBuilder();
+                    p.params.forEach(par ->
+                    {
+                        params.add(par.name, par.value);
+                    });
```

Comment:
```suggestion
                    pub.params.forEach(p-> params.add(p.name, p.value));
```
Use `pub` instead of `s` in `mqttCondition. publishes.forEach`.